product_dimension/i18n/de.po

```diff
@@ -0,0 +1,63 @@
+# Translation of Odoo Server.
```

Comment:
Nice!
But aren't [the other languages available in v9](https://github.com/OCA/product-attribute/tree/9.0/product_dimension/i18n) missing?
Isn't a "OCA Transbot updated translations from Transifex" commit missing, providing the translations done in Transifex?
See https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests#mergesquash-the-commits-generated-by-transifex

Reply:
good point: I created migration based on 10.0 branch and there the i18n folder is missing.. 
https://github.com/OCA/product-attribute/tree/10.0/product_dimension

how can I add the missing files?